clearwater-socket-factory/clearwater_socket_factory.cpp

```diff
@@ -18,24 +18,24 @@
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
 
-const char* MGMT_SOCKET_PATH = "/tmp/clearwater_management_namespace_socket";
-const char* SIGNALING_SOCKET_PATH = "/tmp/clearwater_signaling_namespace_socket";
 const int MAX_PENDING = 5;
 
-const char* DEFAULT_NS_FD_PATH = "/proc/1/ns/net";
-const std::string SIGNALING_NS_FD_DIR = "/var/run/netns/";
-
 const char* LOG_FILENAME = "/var/log/clearwater-socket-factory.log";
 FILE* LOG_FILE = NULL;
+const char *nsstring = NULL;
 
 void write_timestamp(FILE* stream)
```

Comment:
This function shouldn't be called `write_timestamp` any more as it does more than that now.